### ISSUE TITLE
`--put-` raised exceptions

### ISSUE THREAD

**Issue by heavenshell** [AriyaISIHARA/noqalign#4]
`--put-` raised exceptions.
```
$ cat fixtures.py 
# -*- coding: utf-8 -*-
import sys
from os import (
    path,
)
from unittest import TestCase # noqa
```

```
$ cat fixtures.py| python ../noqalign/noqalign/noqalign.py --put- -
Traceback (most recent call last):
  File "../noqalign/noqalign/noqalign.py", line 222, in <module>
    main()
  File "../noqalign/noqalign/noqalign.py", line 218, in main
    Noqalign.commandline(args)
  File "../noqalign/noqalign/noqalign.py", line 95, in commandline
    nql.write(put=put, align=align)
  File "../noqalign/noqalign/noqalign.py", line 53, in write
    col = self._calc_alignment(put, align)
  File "../noqalign/noqalign/noqalign.py", line 63, in _calc_alignment
    for line in self._lines
ValueError: max() arg is an empty sequence
```

**Comment by AriyaISIHARA** [comment]
Thanks for the report.
This feature seems to be a bug that occurs when Noqalign attempts to align `# noqa: F401` comments while there is no `# noqa: F401` comment to align (single `# noqa` comments without `F401`, such as in the last line of the above case, are out of the scope of the present version of Noqalign).
I will add such a testcase and fix it (likely within this month 2017 June).

**Comment by heavenshell** [comment]
Thanks for investigation.
Sure, np for out of the scope.

Looking foward to fix it 😊

**Comment by AriyaISIHARA** [comment]
Now PR #5 has been merged, I hope the bug is fixed in the present version 1.0.1. 🙏
